src/agent/otubus.cpp

```diff
@@ -1198,10 +1224,12 @@ int UbusServer::UbusGetInformation(struct ubus_context *     aContext,
             char                multicastAddr[10] = "ff03::2";
             long                value;
 
+            memset(&mNetworkdataBuf, 0, sizeof(mNetworkdataBuf));
             blob_buf_init(&mNetworkdataBuf, 0);
 
             SuccessOrExit(error = otIp6AddressFromString(multicastAddr, &address));
 
+            // XXX: cannot understand what this magic number refers to
```

Comment:
This is the TLV type value, 5 is Route64 TLV (Thread1.1 spec 5-65), 16 is Child Table TLV (Thread 1.1 spec 10-26), otThreadSendDiagnosticGet() function needs parameters of tlv types.